common/containers/Tabs/Contracts/index.tsx

```diff
@@ -27,7 +27,13 @@ class Contracts extends Component<RouteComponentProps<{}>> {
 
     return (
       <TabSection isUnavailableOffline={true}>
-        <SubTabs tabs={tabs} match={match} location={location} history={history} />
+        <SubTabs
+          tabs={tabs}
+          match={match}
+          location={location}
+          history={history}
+          staticContext={this.props.staticContext}
```

Comment:
This addition of staticContext should be unnecessary, make sure you installed the type definitions using `yarn` as this looks like a typedef / library version mismatch if it's warning you about this.

Reply:
@wbobeirne ok, I'll that. This was just because I got an error and couldn't push

Reply:
@wbobeirne fixed